.github/workflows/build-containers.yml

```diff
@@ -111,80 +152,102 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image: ${{ fromJson(needs.build-image-list.outputs.images) }}
-        series:
-          - name: '23'
-            os: 'el9'
+        include: ${{ fromJson(needs.build-image-list.outputs.matrix) }}
     steps:
-      - name: Push to Harbor (${{ matrix.series }}-development)
-        # FIXME: reenable for OSG 23 development when available
-        if: matrix.series.name != '23'
+      - uses: actions/checkout@v3
+
+      - name: Print raw matrix configuration
+        run: |
+          echo "Raw matrix configuration: ${{ matrix }}"
+      - name: Set environment variables
```

Comment:
This is hard to follow because it's not clear what the input format is (presumably the long config string?). A comment here with an example item from the JSON would be really helpful